### PR TITLE
Freon tanks and adds them to dispensers

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Adds freon tanks and puts them inside some tank dispensers
/:cl:

Also recompiled tgui which fixed the cryo UI as well.

### CODE REVIEW

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 1]
```diff
@@ -179,3 +179,22 @@
 	name = "double emergency oxygen tank"
 	icon_state = "emergency_engi"
 	volume = 10
+
+/*
+ * Freon
+ */
+/obj/item/weapon/tank/internals/freon
+	name = "freon tank"
+	desc = "Contains dangerous plasma. Do not inhale. Warning: extremely flammable."
```
Comment (ghost) [inline code/game/objects/items/weapons/tanks/tank_types.dm]: You had one job

File: code/game/objects/items/weapons/tanks/tank_types.dm [hunk 2]
```diff
@@ -179,3 +179,22 @@
...
+	flags = CONDUCT
+	slot_flags = null	//they have no straps!
+	force = 8
+
+
+/obj/item/weapon/tank/internals/freon/New()
+	..()
+	air_contents.assert_gas("freon")
+	air_contents.gases["freon"][MOLES] = (3*ONE_ATMOSPHERE)*volume/(R_IDEAL_GAS_EQUATION*T20C)
+	air_contents.temperature = 120
```
Comment (ghost) [inline code/game/objects/items/weapons/tanks/tank_types.dm:199]: freon needs to be cold, not hot

**Review by ghost — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/tanks/tank_types.dm (hunk 2 above)
Comment (j-mie) [reply code/game/objects/items/weapons/tanks/tank_types.dm:199]: Isn't Byond temperature in kelvin? So that'd be `-153.15C`

**Comment by j-mie** [comment]
@BTFO Not really relevant to this issue but you were banned for ban evasion ingame right? Ping me on our Discord: https://discord.gg/ncTbx

**Review by j-mie — APPROVED** [review]

**Review by j-mie — APPROVED** [review]

**Comment by ghost** [comment]
@JamieH I was indeed banned by someone for alledged ban evasion, i however never evaded a single ban and have not been banned before, I'll see if i can get on the discord once i get back from work

**Comment by j-mie** [comment]
@BTFO Alright cool, Once you're on ping me and we'll see what we can do about ban, I've sent a new Discord link with 12 hour expiry https://discord.gg/WY9g5 (Ping me if that doesn't work)

**Pull request merged**